docs/extensions/MouseCursor.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Mouse.Cursor attached property
+author: martinsuchan
+ms.date: 11/05/2017
+description: Mouse.Cursor attached property enables you to easily change the mouse cursor over specific Framework elements.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, Mouse, cursor, extensions
+---
+
+# Mouse.Cursor attached property
+
+The **Mouse.Cursor attached property** enables you to easily change the mouse cursor over specific Framework elements.
+
+## Example
+
+First we need to add the namespace declaration, in this case we added to the root Page element:
+```xaml
+xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+```
+
+Then we can use the attached property on any FrameworkElement in XAML:
+```xaml
+extensions:Mouse.Cursor="Hand"
+```
+
+This is how it looks in full Page context:
+```xaml
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MouseCursorPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"    
+    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Border extensions:Mouse.Cursor="Hand"
+	        Width="220" Height="120" Background="DeepSkyBlue"
+	        HorizontalAlignment="Center" VerticalAlignment="Center"/>
+    </Grid>
+</Page>
+```
+
+## Note
```

Comment:
Use the below format for displaying notes as per [#issuecomment-341176267](https://github.com/Microsoft/UWPCommunityToolkit/issues/1569#issuecomment-341176267)

```
> [!NOTE]
Some note
```

Reply:
Fixed